com.reprezen.swagedit/src/com/reprezen/swagedit/preferences/SwaggerValidationPreferences.java

```diff
@@ -31,6 +33,7 @@
         implements IWorkbenchPreferencePage, IPropertyChangeListener {
 
     public SwaggerValidationPreferences() {
+        // GRID is needed because we are not attaching the editor fields directly to FieldEditorParent, but to its child
```

Comment:
@tfesenko , @andylowry , BTW, if we were to start this over again, I might have suggested using a checked listbox instead of a group of static listbox controls.  Only because it makes it easier (I think) to add new contexts. 

If you think the edge cases, where the current layout might break, warrant a fix, consider using a checked list as a possible alternative.  But at this point, please choose whichever fix is easiest.  (Or choose no fix at all, if you think we're OK with the current implementation.)